Cargo.toml

```diff
@@ -23,9 +23,17 @@ members = [
 ]
 
 [profile.release]
+codegen-units = 1
 lto = "fat"
 panic = "unwind"
 
+# Defaults for `release` profiles as specified in https://doc.rust-lang.org/cargo/reference/profiles.html#release
+[profile.release-unoptimized]
```

Comment:
Should we also add the debug = true field here? Would make it easier to debug potential compile error

Reply:
Forget it. it would slow down the compile process